test/unit/ngsiv2/expressions/jexlBasedTransformations-test.js

```diff
@@ -141,6 +141,26 @@ const iotAgentConfig = {
                 }
             ]
         },
+        WeatherStationWithIdNumber: {
+            commands: [],
+            type: 'WeatherStation',
+            entityNameExp: 'id',
+            lazy: [],
+            /* info which will be part of typeInformation passed to iotalib.update*/
+            service: 'smartgondor',
+            subservice: 'gardens',
+            name: '1234',
+            id: '1234',
+            /* end info */
```

Comment:
is this required?

Reply:
Really service, subservice, id and type are expected in typeInformation received by iotagent.update function

Reply:
Test updated in https://github.com/telefonicaid/iotagent-node-lib/pull/1484/commits/34fea35a65c215d6b5f58d2059070c86ba7d42b3